src/Persistence/Fake/FakeResetPasswordInternalRepository.php

```diff
@@ -60,4 +60,9 @@ public function removeExpiredResetPasswordRequests(): int
     {
         throw new FakeRepositoryException();
     }
+
+    public function removeRequests(?object $user = null): void
```

Comment:
I’m not sure we need the nullable case where it deletes everything. I can’t think of a use-case